k9mail/src/main/java/com/fsck/k9/Account.java

```diff
@@ -1615,6 +1628,22 @@ public void setRemoteSearchNumResults(int val) {
         mRemoteSearchNumResults = (val >= 0 ? val : 0);
     }
 
+    public boolean getResizeEnabled() {
```

Comment:
consider `getImageResizeEnabled` to make it more obvious what this is for (applies to all 4 methods).